etools-searchable-multiselection-menu.html

```diff
@@ -149,6 +149,11 @@
         color: rgba(0,0,0,0.54);
         background-color: #EEEEEE;
       }
+      .emptyValue {
```

Comment:
This will override the css from `.list-wrapper paper-item`. 
Can we change `-- None --` to `None` and align it with the other options values?